index.js

```diff
@@ -0,0 +1,68 @@
+const validateSingleTeamRule = teamIds => (
+  Object.values(teamIds).reduce((valid, teamCount) => (
+    valid && teamCount <= 2
+  ), true)
+)
+
+const validateSingleGameRule = gameIds => (
+  Object.values(gameIds).reduce((valid, gameCount) => (
+    valid && gameCount <= 3
+  ), true)
+)
+
+const validatePositionCount = positions => (
+  Object.values(positions).reduce((valid, position) => (
+    valid && position === 0
+  ), true)
+)
```

Comment:
`every` would be a better choice here since it checks that every item in the list is true which is what you want.